packages/jazz-tools/src/tools/subscribe/SubscriptionScope.ts

```diff
@@ -341,16 +341,14 @@ export class SubscriptionScope<D extends CoValue> {
       this.resolve = {};
     }
 
-    if (this.resolve.$each || key in this.resolve) {
-      return;
-    }
```

Comment:
This method is accessed when the child load haven't been tried yet.

By returning early here we were preventing the property access to trigger the load of the new child id.